rfcs/0000-data-structures/0000-data-structures.md

```diff
@@ -0,0 +1,265 @@
+| :--------- | :--------- | :----------------------------------------------------------- |
+| `capacity` | uint64     | **The size of the cell.** When a new cell is generated (via transaction), one of the verification rule is `capacity ≥ len(capacity)+len(data)+len(type)+len(lock)`. This value also represents the balance of CKB coin, just like the `balance` field in the Bitcoin's UTXO. (E.g. Alice owns 100 CKB coins means she can unlock a group of cells that has 100 amount of `capacity` in total.) |
+| `data`     | Bytes      | **Arbitrary data.** This part is for storing states or scripts.  In order to make this cell valid on-chain, the data filled in this field should comply with the logics and rules defined by `type` or `lock`. |
+| `type`     | `Script`   | **A Script that defines the type of the cell.** In a transaction, if an input cell and an output cell has the same `type` field, then the `data` part of these two cells is limited by the `type` script upon the transaction verification. (I.e. `type` is a script that limits how the `data` field of the new cells can be changed from the old cells.) `type` is required to has a data structure of `script`. **This field is optional.** |
+| `lock`     | `Script`   | **A Script that defines the ownership of the cell**, just like the `lock` field in the Bitcoin's UTXO. Whoever can provide unlock arguments that makes the execution of this script success can use this cell as input in an transaction (i.e. has the ownership of this cell). This is a more general version of P2SH scheme in Bitcoin. |
+
+
+
+More information about Cell can be found in the [whitepaper](https://github.com/nervosnetwork/rfcs/blob/master/rfcs/0002-ckb/0002-ckb.md#42-cell).
+
+
+
+## Script
+
+### Example
+
+```json
+{
+  "version": 0,
+  "binary_hash": "0x12b464bcab8f55822501cdb91ea35ea707d72ec970363972388a0c49b94d377c",
+  "args": [
+    "3044022038f282cffdd26e2a050d7779ddc29be81a7e2f8a73706d2b7a6fde8a78e950ee0220538657b4c01be3e77827a82e92d33a923e864c55b88fd18cd5e5b25597432e9b",
+    "1"
+  ]
+}
+```
+
+
+
+### Description
+
+| Name          | Type       | Description                                                  |
+| :------------ | :--------- | :----------------------------------------------------------- |
+| `version`     | uint8      | **The version of the script.** It‘s used to distinguish transactions when there's a fork happened to the blockchain system. |
+| `binary_hash` | H256(hash) | **The hash of ELF formatted binary that contains an RISC-V based script. For space consideration, the actual script is either attached to current transaction as a dep cell, or included in as an embed field in current the transaction. In either case, the hash of actual binary(which is cell data part for a dep cell, and the content of the whole field for an embed field) should match binary_hash included here. The actual binary is loaded into an CKB-VM instance when they are specified upon the transaction verification. |
```

Comment:
missing ** at the end of the first sentence

Reply:
> For space consideration, the actual script is either attached to current transaction as a dep cell, or included in as an embed field in current the transaction.

typo

-> For **space efficiency consideration**, the actual script is either attached to the current transaction as a dep cell, or included in as an embed field **in the current transaction**.